src/GitHubCalendar.js

```diff
@@ -3,21 +3,80 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import CalendarHeatmap from 'react-calendar-heatmap';
+import ReactTooltip from 'react-tooltip';
+import Legends from './Legends';
 
-const Wrapper = styled.div``;
+import '../static/css/style.css';
+
+const Wrapper = styled.div`
+    overflow-x: scroll;
+    direction: rtl;
+`;
 
 class GitHubCalendar extends React.Component {
     constructor(props) {
         super(props);
-
-        this.state = {};
     }
 
+    countList = this.props.data.map((value) => value.count);
+    largest = Math.max.apply(null, this.countList);
```

Comment:
Please move this code to the inside of render function.

Reply:
I applied it!